scripts/pre_push_hook.py

```diff
@@ -122,10 +146,10 @@ def _collect_files_being_pushed(ref_list, remote):
         ref_list: list of references to parse (provided by git in stdin)
         remote: the remote being pushed to
     Returns:
```

Comment:
Ah, I got it -- I didn't realize you could push more than one branch at once, and in fact the command we usually use is "git push origin branch-name" rather than "git push REMOTE --all". So my intended question was more along the lines of why we couldn't just lint "branch-name" rather than everything that was being pushed (though I suppose that is what we would usually be doing anyway, since usually only one branch is being pushed).

So overall I think this is probably fine, but you might want to add a comment to the for loop explaining why it is possible that more than one branch can occur in collected_files.